src/main/resources/application.yml

```diff
@@ -9,4 +9,6 @@ cloud:
 
 application:
   bucket:
-    name: my-aws-demo-app-images-bucket
\ No newline at end of file
+    name: my-aws-demo-app-images-bucket
+  queue:
+    uri: https://sqs.ap-northeast-1.amazonaws.com/039316342888/my-awesome-sqs
```

Comment:
Created the SQS in amazon and modified IAM Rule to allow privileges to have full access to SQS